lightning/pkg/importer/import.go

```diff
@@ -433,13 +434,31 @@ func NewImportControllerWithPauser(
 		if err != nil {
 			return nil, err
 		}
+	case config.BackendRemote:
```

Comment:
What error will occur if the user configures the remote backend? Should we prompt the user that this feature is not yet implemented? Additionally, should we add a description in the code comments to avoid confusion for users and developers?